examples/listen-to-any-until-condition.yaml

```diff
@@ -0,0 +1,11 @@
+document:
+  dsl: '1.0.0-alpha1'
+  namespace: test
+  name: listen-to-any
+  version: '0.1.0'
+do:
+  - callDoctor:
+      listen:
+        to:
+          any: []
```

Comment:
Ideally we should test the schema by adding a failing example (under .ci/validation/test/fixtures/failing) where `any` is empty and `until` missing